core/object/worker_thread_pool.cpp

```diff
@@ -701,6 +701,8 @@ void WorkerThreadPool::init(int p_thread_count, float p_low_priority_task_ratio)
 
 	max_low_priority_threads = CLAMP(p_thread_count * p_low_priority_task_ratio, 1, p_thread_count - 1);
 
+	print_line(vformat("WorkerThreadPool: %d threads, %d max low-priority.", p_thread_count, max_low_priority_threads));
```

Comment:
That seems a bit noisy to print on every single engine start, even when users don't know some of the stuff they're doing may rely on WTP.

`print_verbose` sounds more appropriate for this kind of info message.